### PR TITLE
Docs: Improve Structure and Wording

### PR DESCRIPTION
**Pull request by dakanji**

### CODE REVIEW

File: Docs/Configuration.tex [hunk 1]
```diff
@@ -2230,8 +2229,8 @@ \subsection{Quirks Properties}\label{kernelpropsquirks}
     \item Download \href{https://github.com/LongSoft/UEFITool/releases}{UEFITool} and
       \href{https://github.com/LongSoft/Universal-IFR-Extractor/releases}{IFR-Extractor}.
     \item Open the firmware image in UEFITool and find \texttt{CFG Lock} unicode string.
-      If it is not present, the firmware may not have this option and one should stop.
-    \item Extract the \texttt{Setup.bin} PE32 Image Section (the one UEFITool found) through
+      If it is not present, the firmware may not have this option and the proccess therefore be discontinued.
```
Comment (PMheart) [inline Docs/Configuration.tex:2232]: the process *should* therefore be discontinued I guess?

File: Docs/Configuration.tex [hunk 2]
```diff
@@ -112,31 +112,31 @@ \section{Introduction}\label{introduction}
...
-configuration file format used to setup the correct functioning of macOS
+configuration file format used to setup the correct functioning of the macOS
 operating system. It is to be read as the official clarification of expected
 OpenCore behaviour. All deviations, if found in published OpenCore releases,
-shall be considered documentation or implementation bugs, and are requested to be
-reported through \href{https://github.com/acidanthera/bugtracker}{Acidanthera Bugtracker}.
-Errata sheet is available in
+shall be considered to be documentation or implementation bugs which should be
+reported via the \href{https://github.com/acidanthera/bugtracker}{Acidanthera Bugtracker}.
+An errata sheet is available in
```
Comment (PMheart) [inline Docs/Configuration.tex:120]: Errata is the plural form or erratum, so *Errata sheet* should be fine.

File: Docs/Configuration.tex [hunk 3]
```diff
@@ -112,31 +112,31 @@ \section{Introduction}\label{introduction}
...
 This document provides information on
 \href{https://github.com/acidanthera/OpenCorePkg}{OpenCore} user
-configuration file format used to setup the correct functioning of macOS
+configuration file format used to setup the correct functioning of the macOS
 operating system. It is to be read as the official clarification of expected
 OpenCore behaviour. All deviations, if found in published OpenCore releases,
-shall be considered documentation or implementation bugs, and are requested to be
-reported through \href{https://github.com/acidanthera/bugtracker}{Acidanthera Bugtracker}.
-Errata sheet is available in
+shall be considered to be documentation or implementation bugs which should be
```
Comment (PMheart) [inline Docs/Configuration.tex:118]: How about *bugs of documentation or implementation*?

File: Docs/Configuration.tex [hunk 4]
```diff
@@ -112,31 +112,31 @@ \section{Introduction}\label{introduction}
...
+reported via the \href{https://github.com/acidanthera/bugtracker}{Acidanthera Bugtracker}.
+An errata sheet is available in
 \href{https://github.com/acidanthera/OpenCorePkg/blob/master/Docs/Errata/Errata.pdf}{OpenCorePkg repository}.
 
-This document is structured as a specification, and is not meant to provide a step by
-step algorithm for configuring end-user board support package (BSP). The intended audience
-of the document are programmers and engineers with basic understanding of macOS internals
-and UEFI functioning. For these reasons this document is available exclusively in English,
+This document is structured as a specification and is not meant to provide a step by
+step guide to configuring an end-user Board Support Package (BSP). The intended audience
```
Comment (PMheart) [inline Docs/Configuration.tex:124]: *step-by-step* maybe?

File: Docs/Configuration.tex [hunk 5]
```diff
@@ -2549,9 +2548,9 @@ \subsection{Scheme Properties}\label{kernelpropsscheme}
   \texttt{Mkext}, \texttt{Prelinked}) when available.
 
   Different variants of macOS support different kernel caching variants designed to improve
-  boot performance. This setting allows to prevent using faster kernel caching variants
-  if slower variants are available for debugging and stability reasons. I.e. by
-  specifying \texttt{Mkext} one will disable \texttt{Prelinked} for e.g. 10.6 but not 10.7.
+  boot performance. This setting prevents the use of faster kernel caching variants
+  if slower variants are available for debugging and stability reasons. I.E., by
```
Comment (vit9696) [inline Docs/Configuration.tex:2552]: I think it should be `I.e.`, not `I.E.`. Some [websites](https://ell.stackexchange.com/questions/56861/capitalize-i-e-and-e-g) agree at least.

File: Docs/Configuration.tex [hunk 6]
```diff
@@ -112,31 +112,31 @@ \section{Introduction}\label{introduction}
...
+An errata sheet is available in
 \href{https://github.com/acidanthera/OpenCorePkg/blob/master/Docs/Errata/Errata.pdf}{OpenCorePkg repository}.
 
-This document is structured as a specification, and is not meant to provide a step by
-step algorithm for configuring end-user board support package (BSP). The intended audience
-of the document are programmers and engineers with basic understanding of macOS internals
-and UEFI functioning. For these reasons this document is available exclusively in English,
+This document is structured as a specification and is not meant to provide a step by
+step guide to configuring an end-user Board Support Package (BSP). The intended audience
+of the document are programmers and engineers with a basic understanding of macOS internals
```
Comment (PMheart) [inline Docs/Configuration.tex:125]: ...of the documents *is*....

(The subject of this sentence is audience, which has single form)

File: Docs/Configuration.tex (hunk 2 above)
Comment (vit9696) [reply Docs/Configuration.tex:120]: I think an article refers to the noun (sheet) here with errata being treated as an adjective. So either "an errata sheet" or "errata sheets", but I believe the former is more common. Omitting an article in such cases is probably allowed for the manual speech, but better to have it. So I believe the change is correct.

File: Docs/Configuration.tex (hunk 3 above)
Comment (vit9696) [reply Docs/Configuration.tex:118]: Whatever is more idiomatic, not a native speaker, but I thought "documentation or implementation bugs" is more common.

File: Docs/Configuration.tex [hunk 7]
```diff
@@ -574,10 +574,10 @@ \subsection{Installation and Upgrade}\label{configuration-install}
 
 For BIOS booting a third-party UEFI environment provider will have to
 be used. \texttt{OpenDuetPkg} is one of the known UEFI environment providers
-for legacy systems. To run OpenCore on such a legacy system one can install
-\texttt{OpenDuetPkg} with a dedicated tool --- BootInstall (bundled with OpenCore).
+for legacy systems. To run OpenCore on such a legacy system, \texttt{OpenDuetPkg}
+can be installed with a dedicated tool --- BootInstall (bundled with OpenCore).
 \href{https://github.com/corpnewt/gibMacOS}{Third-party utilities} can be used to
-perform this on systems different from macOS.
+perform this on non-macOS systems.
```
Comment (PMheart) [inline Docs/Configuration.tex:580]: *on systems other than macOS*?

File: Docs/Configuration.tex (hunk 1 above)
Comment (dakanji) [reply Docs/Configuration.tex:2232]: Noted

File: Docs/Configuration.tex (hunk 3 above)
Comment (dakanji) [reply Docs/Configuration.tex:118]: Doesn't sound right

File: Docs/Configuration.tex (hunk 4 above)
Comment (dakanji) [reply Docs/Configuration.tex:124]: Noted

File: Docs/Configuration.tex [hunk 8]
```diff
@@ -1193,8 +1193,8 @@ \subsection{Introduction}\label{booterintro}
 \item \texttt{Above 4G Decoding} or similar enabled in firmware
   settings if present. Note, that on some motherboards (notably ASUS WS-X299-PRO) this
   option causes adverse effects, and must be disabled. While no other motherboards
-  with the same issue are known, consider this option to be first to check if one
-  encounters erratic boot failures.
+  with the same issue are known, this option should be checked first whenever erratic boot
```
Comment (PMheart) [inline Docs/Configuration.tex:1196]: *whenever encountering boot failure*?

(This is sort of subject, and also failure is uncountable)

File: Docs/Configuration.tex [hunk 9]
```diff
@@ -1329,19 +1329,18 @@ \subsection{Quirks Properties}\label{booterpropsquirks}
   \textbf{Failsafe}: \texttt{false}\\
   \textbf{Description}: Disable single user mode.
 
-  This is a security option allowing one to restrict single user mode usage
+  This is a security option that restricts activating single user mode
```
Comment (PMheart) [inline Docs/Configuration.tex:1332]: restricts *the activation of*...

File: Docs/Configuration.tex [hunk 10]
```diff
@@ -1495,7 +1494,7 @@ \subsection{Quirks Properties}\label{booterpropsquirks}
   into the unavailable range. When \texttt{AppleDebug} is enabled, usually the
   debug log may contain messages like \texttt{AAPL: [EB|`LD:LKC] \} Err(0x9)}.
   To find the optimal value, manually append \texttt{slide=X} to \texttt{boot-args}
-  and log the largest one that won't cause boot failure.
+  and log the largest one that will not result in boot failures.
```
Comment (PMheart) [inline Docs/Configuration.tex:1497]: Still, failure is uncountable.

File: Docs/Configuration.tex (hunk 5 above)
Comment (dakanji) [reply Docs/Configuration.tex:2552]: Noted

File: Docs/Configuration.tex (hunk 6 above)
Comment (dakanji) [reply Docs/Configuration.tex:125]: I don't think you can use "is" for "programmers" and "engineers"

File: Docs/Configuration.tex [hunk 11]
```diff
@@ -2216,9 +2215,9 @@ \subsection{Quirks Properties}\label{kernelpropsquirks}
   MSR modification in AppleIntelCPUPowerManagement.kext, commonly causing early
   kernel panic, when it is locked from writing.
 
-  Certain firmwares lock \texttt{PKG\_CST\_CONFIG\_CONTROL} MSR register. To check its
-  state one can use bundled \texttt{VerifyMsrE2} tool. Select firmwares have this
-  register locked on some cores only.
+  Certain firmwares lock \texttt{PKG\_CST\_CONFIG\_CONTROL} MSR register. The bundled
+  \texttt{VerifyMsrE2} tool can be used to check its state. Some firmware have this
+  register locked only on some cores.
```
Comment (PMheart) [inline Docs/Configuration.tex:2220]: *merely* can be more academic than *only*?

File: Docs/Configuration.tex (hunk 7 above)
Comment (dakanji) [reply Docs/Configuration.tex:580]: Noted

File: Docs/Configuration.tex (hunk 9 above)
Comment (dakanji) [reply Docs/Configuration.tex:1332]: Think current is equally valid but will change

File: Docs/Configuration.tex (hunk 8 above)
Comment (dakanji) [reply Docs/Configuration.tex:1196]: Actually, it is.

File: Docs/Configuration.tex (hunk 6 above)
Comment (PMheart) [reply Docs/Configuration.tex:125]: Hmm, how about *The target audience is those with a basic understanding of*...

File: Docs/Configuration.tex (hunk 10 above)
Comment (dakanji) [reply Docs/Configuration.tex:1497]: As earlier response

File: Docs/Configuration.tex (hunk 8 above)
Comment (PMheart) [reply Docs/Configuration.tex:1196]: Fun... The Chinese-English/Japanese-English dictionary told me that it is uncountable when interpreted as lack of success, yet the Oxford English dictionary said that failure is countable. @_@

File: Docs/Configuration.tex (hunk 10 above)
Comment (PMheart) [reply Docs/Configuration.tex:1497]: Right. Then it can be left either way I guess, and I am fine with it. :)

File: Docs/Configuration.tex [hunk 12]
```diff
@@ -3288,12 +3287,12 @@ \subsection{Security Properties}\label{miscsecurityprops}
   \href{https://support.apple.com/en-us/HT208330}{\texttt{Full Security}} of Apple
   Secure Boot.
 
-  To start using personalised Apple Secure Boot one will have to reinstall the
-  operating system or personalise it. Until the operating system is personalised
-  one will only be able to load macOS DMG recovery. If DMG recovery is missing,
+  To start using personalised Apple Secure Boot, the  operating system will have
```
Comment (PMheart) [inline Docs/Configuration.tex:3290]: Please remove the extra space between *the* and *operating*.

File: Docs/Configuration.tex [hunk 13]
```diff
@@ -3288,12 +3287,12 @@ \subsection{Security Properties}\label{miscsecurityprops}
...
+  to be reinstalled or personalised. Unless the operating system is personalised,
```
Comment (PMheart) [inline Docs/Configuration.tex:3291]: It sounds a bit wordy here. How about just *Unless personalised*?

File: Docs/Configuration.tex [hunk 14]
```diff
@@ -3653,16 +3651,16 @@ \subsection{Security Properties}\label{miscsecurityprops}
...
   \texttt{SecureBootModel} to any valid value but \texttt{Disabled} is equivalent to
   \href{https://support.apple.com/en-us/HT208330}{\texttt{Medium Security}}
-  of Apple Secure Boot. To achieve \texttt{Full Security} one will need to
-  also specify \texttt{ApECID} value.
+  of Apple Secure Boot. The \texttt{ApECID} value must also be specified to
+  achieve \texttt{Full Security}.
 
   Enabling Apple Secure Boot is more demanding to incorrect configurations,
-  buggy macOS installations, and unsupported setups. Things to keep in mind:
+  buggy macOS installations, and unsupported setups. Things to consider:
```
Comment (PMheart) [inline Docs/Configuration.tex:3658]: *keep in mind* should be fine here I guess? (i.e. to put emphasis)

File: Docs/Configuration.tex [hunk 15]
```diff
@@ -3653,16 +3651,16 @@ \subsection{Security Properties}\label{miscsecurityprops}
...
 
   \begin{enumerate}
     \tightlist
-    \item Just like on T2 Macs one will not be able to install any unsigned kernel
-      drivers and several signed kernel drivers including NVIDIA Web Drivers.
+    \item As with T2 Macs, unsigned kernel drivers and several signed kernel
```
Comment (PMheart) [inline Docs/Configuration.tex:3662]: just *With T2 Macs* maybe?

File: Docs/Configuration.tex (hunk 12 above)
Comment (dakanji) [reply Docs/Configuration.tex:3290]: Noted

**Comment by PMheart** [comment]
Hello! I have taken a quick review and put my comments there. Please also run `BuildDocs.tool` after everything is done.

File: Docs/Configuration.tex (hunk 3 above)
Comment (PMheart) [reply Docs/Configuration.tex:118]: Maybe *mistakes in documentation or bugs of implementation*? Sounds wordy though...

Comment (vit9696) [reply Docs/Configuration.tex:118]: In my opinion, leave it is as is.

File: Docs/Configuration.tex (hunk 6 above)
Comment (vit9696) [reply Docs/Configuration.tex:125]: The shorter the better, the `is anticipated` way might actually be more idiomatic.

File: Docs/Configuration.tex (hunk 3 above)
Comment (PMheart) [reply Docs/Configuration.tex:118]: Right. Marked as resolved.

**Comment by vit9696** [comment]
As for BuildDocs — could do ourselves if you do not have TeX installed.

File: Docs/Configuration.tex (hunk 15 above)
Comment (dakanji) [reply Docs/Configuration.tex:3662]: Don't think so. This is saying the behaviour is like it is on T2 Macs.

Comment (PMheart) [reply Docs/Configuration.tex:3662]: Makes sense. Sorry for confusion.

File: Docs/Configuration.tex (hunk 14 above)
Comment (dakanji) [reply Docs/Configuration.tex:3658]: Don't think so.

File: Docs/Configuration.tex (hunk 13 above)
Comment (dakanji) [reply Docs/Configuration.tex:3291]: Repeating "OS" provides clarity and don't think it is particularly wordier than what applies across the rest of the document.

Comment (PMheart) [reply Docs/Configuration.tex:3291]: I would try to avoid repetitions where available. How about *Until the completion of personalisation,*?

File: Docs/Configuration.tex (hunk 14 above)
Comment (vit9696) [reply Docs/Configuration.tex:3658]: Yeah, I've sort of overdone with the informal things. Not to be blamed for too much for writing the whole thing though.

File: Docs/Configuration.tex (hunk 11 above)
Comment (dakanji) [reply Docs/Configuration.tex:2220]: Explain "merely"?
Changes "locked on some cores only" to "locked only on some cores". I.e., moves "only" to correct position.

Or do you mean to use "merely" in place of "only" which would be wrong as "merely" connotes triviality while "only" connotes singularity which I think is the intention here.

Comment (PMheart) [reply Docs/Configuration.tex:2220]: Thanks for the explanation! I am not a native English speaker, though I kind of remembered such thesauruses in some academic writing classes... @_@

**Comment by PMheart** [comment]
Thanks!
